components/src/core/components/CardTable/Table/table-header-sort-dropdown.scss

```diff
@@ -24,7 +24,7 @@
 
     ul {
       list-style: none;
-      padding: $oxd-dropdown-padding;
+      padding: $oxd-dropdown-ul-padding;
```

Comment:
This need to be discussed with Mike

Reply:
Confirmed with Mike. He's ok with the changed style. Asked to continue with goal list.